pairing-tool/index.js

```diff
@@ -67,6 +71,7 @@ async function runPairingTool() {
 
     console.log('Welcome to the Brave Button pairing tool.')
 
+    // eslint-disable-next-line no-constant-condition -- we want to implement a run loop here
```

Comment:
What do you mean by a run loop, @schwarrrtz ?
`for(;;)` should have the same behaviour as `while(true)`.

Reply:
A run loop is a concept from microcontroller programming which is basically just `while(true) { do stuff } `

I'm not familiar with `for(;;)` - I wasn't aware that it has the same behaviour as `while(true)`. I feel like `while(true)` is more clear. I mentioned this in passing to Richard and he seemed to agree with me.

Reply:
I'm cool with that. Thanks for clarifying.